_build/objectEditingOutput/WurstExportedObjects_w3a.wurst.txt

```diff
@@ -1,778 +0,0 @@
-package WurstExportedObjects_w3a
```

Comment:
As discussed, I won't remove the WurstExportedObjects files yet.

Reply:
as discussed, I don't care

Reply:
Do you mean that you won't fix the PR to not do what I won't accept?